src/movie-components/movie-container-component/_movie-container.scss

```diff
@@ -5,3 +5,32 @@
   justify-content: center;
   width: 100vw;
 }
+
+.loading {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: #f6f6f6;
+  border: 1px solid black;
+  height: 250px;
+  width: 250px;
+  text-align: center;
+  overflow: hidden;
+  margin-left: 50%;
+  transform: translateX(-50%);
```

Comment:
I like the implementation of  animation here, bringing some fanciness to our page!